### DISCUSSION TITLE
Github Discussions and Teams

### DISCUSSION THREAD

**Discussion by YesThatAllen** [community/community#21953] [New to GitHub]
<p>I’m pretty interested in <a href="https://github.com/blog/2471-introducing-team-discussions" rel="nofollow noopener">Github Discussions</a>, however, I have a problem with the Teams.</p>
<p>I have a number of 3rd parties we work with, and have made Teams for them. If I leave them as-is, they can see Discussions which we don’t make Private.</p>
<p>If I convert the users to external collaborators, the penalty is pretty steep:</p>
<p><em>Converting members to outside collaborators will remove them from this organization and from all teams.</em></p>
<p><em>Their repository access will be preserved by making them collaborators on all repositories that their teams gave them access to. They will retain access to repositories that they were previously collaborators on, but all other access to this organization’s repositories will be lost.</em></p>
<p>It means I’ll loose track of who’s from which development partner of ours, and they may lose access to repos.</p>
<p>Has anyone else run across this? What are you doing about it?</p>

**Comment by YesThatAllen** [comment]
<p>At this point, we’re basically not using Discussions bc it’s too easy to leave a post public, which is a drag because I think it’s a great feauture.</p>
<p>Bumping this message to see what others are doing.</p>
<p><a href="https://github.com/blog/2471-introducing-team-discussions" rel="nofollow noopener">https://github.com/blog/2471-introducing-team-discussions</a></p>

**Reply by gjsjohnmurray** [reply]
I share this concern. I wish that part of the definition of a Team could set that a new discussion is Private by default, rather than being visible to all organization members (the current default):

![image](https://user-images.githubusercontent.com/6726799/193810236-deb6142f-e385-42b1-b3fd-8f03da42b8ac.png)

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/yesthatallen">@yesthatallen</a> I wanted to comment here and just acknowledge the difficulty you’re having with 3rd parties on your Teams and the privacy of Discussions.</p>
<p>Though we don’t have a built in way to do this at this time, I want to let you know that we’re passing this feedback on to the appropriate teams for future consideration on newer iterations of the Discussions feature in the future. </p>
<p>Thank you for being a part of the GitHub Community Forum; we’re glad you’re here.</p>
<p>Cheers!</p>
